arches_component_lab/views/api/relatable_resources.py

```diff
@@ -34,44 +34,35 @@ def get(self, request, graph, node_alias):
             "name", "graphid"
         )
 
-        resources = ResourceInstance.objects.filter(graph_id__in=graphs).annotate(
-            order_field=Case(
-                When(resourceinstanceid__in=initial_values, then=Value(0)),
-                default=Value(1),
-            )
+        resources = ResourceInstance.objects.filter(graph_id__in=graphs).exclude(
+            resourceinstanceid__in=initial_values
```

Comment:
Are we sure we want to exclude these? Seems a little surprising, what if some of the initial values don't match the searched term? How to filter them out?